### PR TITLE
{Doc} Add more examples and install extensions part for airgapped cloud guide

### PR DESCRIPTION
**Pull request by fengzhou-msft**
**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: doc/use_cli_in_airgapped_clouds.md [hunk 1]
```diff
@@ -4,33 +4,66 @@
...
 * `az find` to find command examples.
 * Command recommendations based on Aladdin service when the command cannot be parsed correctly.
 * Commands that install another tool such as `az aks install-cli`, `az acr helm install-cli`, `az storage copy`.
 * External channel operations in `az bot` are not available such as Facebook and Wechat.
 * Some links in help messages or error messages cannot be accessible.
 
-## Install
-We are working on solutions to make the installation and upgrade of Azure CLI easier in airgapped clouds. Before that is ready, you need to download the Azure CLI package with public internet access, copy it to the airgapped cloud environment and then install it locally in the airgapped cloud.
+## Install Azure CLI
+We are working on solutions to make the installation and upgrade of Azure CLI easier in airgapped clouds. Before that is ready, you need to download the Azure CLI package with public internet access, copy it to the airgapped cloud environment and then install it locally in the airgapped cloud. You can view all released versions in [Releases](https://github.com/Azure/azure-cli/releases).
```
Comment (yungezz) [inline doc/use_cli_in_airgapped_clouds.md:16]: The top one is latest release, such as `Azure CLI 2.15.1`

**Comment by yonzhan** [comment]
Doc

File: doc/use_cli_in_airgapped_clouds.md [hunk 2]
```diff
@@ -4,33 +4,66 @@
...
-DEB | https://packages.microsoft.com/repos/azure-cli/pool/main/a/azure-cli/ | dpkg -i azure-cli_*.deb
-RPM | https://packages.microsoft.com/yumrepos/azure-cli/ | rpm -ivh --nodeps azure-cli-*.rpm
-MSI | https://azurecliprod.blob.core.windows.net/msi/azure-cli-2.x.x.msi | Start-Process msiexec.exe -Wait -ArgumentList '/I azure-cli-2.x.x.msi'  
+DEB | https://packages.microsoft.com/repos/azure-cli/pool/main/a/azure-cli/ | dpkg -i azure-cli_<version\>-1~<distro\>_all.deb
+RPM | https://packages.microsoft.com/yumrepos/azure-cli/ | rpm -ivh --nodeps azure-cli-<version\>-*.rpm
+MSI | https://azurecliprod.blob.core.windows.net/msi/azure-cli-<version\>.msi | Start-Process msiexec.exe -Wait -ArgumentList '/I azure-cli-<version\>.msi'  
 
-**Note**: The CLI RPM package depends on a `python3` package and you'll need to install it separately while DEB and MSI packages already have a bundled Python in them.
+**Note**:
+1. Replace `<version>` with the actual CLI version you need to use. On Ubuntu/Debian, replace `<distro>` with the result of the `lsb_release -cs` command.
```
Comment (yungezz) [inline doc/use_cli_in_airgapped_clouds.md:25]:
```suggestion
1. Replace `<version>` with the actual CLI version you need to use, such as `2.15.1`. On Ubuntu/Debian, replace `<distro>` with the result of the `lsb_release -cs` command, such as xxx.
```

File: doc/use_cli_in_airgapped_clouds.md [hunk 3]
```diff
@@ -4,33 +4,66 @@
...
-If you are working in an Azure AirGapped Cloud, you should be able to get a cloud metadata URL from its documentation. You can set the environment variable `ARM_CLOUD_METADATA_URL` to this URL, then CLI will load the available clouds and the corresponding cloud endpoints from the URL. The first cloud in the available cloud list will be set as the active cloud by default if the public `AzureCloud` is (most likely) not available.
+If you are working in an Azure AirGapped Cloud, you should be able to get a cloud metadata URL from its documentation. You can set the environment variable `ARM_CLOUD_METADATA_URL` to this URL with:
+```bash
+export ARM_CLOUD_METADATA_URL=https://example.url.com/metadata/endpoints?api-version=2019-05-01
+```
+on Linux shells or
+```powershell
+$Env:ARM_CLOUD_METADATA_URL = "https://example.url.com/metadata/endpoints?api-version=2019-05-01"
+```
+on Windows Powershell.
```
Comment (yungezz) [inline doc/use_cli_in_airgapped_clouds.md:64]: An example metadata file is https://management.azure.com/metadata/endpoints?api-version=2019-05-01

**Review by dbradish-microsoft — CHANGES_REQUESTED** [review]
@fengzhou-msft , if airgapped clouds are going to be a growing trend, would you be okay with this information going into azure-docs-cli probably in the TOC under How-to guides?  We have similar information already over there, such as Work with Multiple Clouds.  Thoughts?

**Review by qwordy — APPROVED** [review]

**Comment by fengzhou-msft** [comment]
@dbradish-microsoft this doc is for temporary workarounds in airgapped clouds under current situation. We will make improvements and when things are easier and more stable, we can show the content in ms doc as well.

**Review by yungezz — APPROVED** [review]

**Pull request merged**